compiler/mio-tflite/CMakeLists.txt

```diff
@@ -4,20 +4,17 @@ if(NOT FlatBuffers_FOUND)
   return()
 endif(NOT FlatBuffers_FOUND)
 
-# TODO recover official release version
-# nnas_find_package(TensorFlowSource EXACT 2.2.0 QUIET)
-#
-# if(NOT TensorFlowSource_FOUND)
-#   return()
-# endif(NOT TensorFlowSource_FOUND)
+# NOTE we cannot use version number like 2.3.0-rc0
+#      use 2.2.90 as workaround for 2.3.0-rc0
+nnas_find_package(TensorFlowSource EXACT 2.2.90 QUIET)
```

Comment:
(Optional) You may use TensorFlowSource-2.3.0-rc0 as a package name ;-)

Reply:
cmake fails with I use `2.3.0-rc0`...

Reply:
this is the error message that I get
```
CMake Error at /.../infra/nncc/CMakeLists.txt:43 (find_package):
  find_package called with invalid argument "2.3.0-rc0"
```

version: `cmake version 3.16.3`